x-pack/plugins/security_solution/public/management/components/artifact_entry_card/artifact_entry_card.tsx

```diff
@@ -37,6 +37,8 @@ export interface CommonArtifactEntryCardProps extends CommonProps {
    */
   policies?: MenuItemPropsByPolicyId;
   loadingPoliciesList?: boolean;
+  // Artifact specific decoration to be displayed in the card section
```

Comment:
Can I suggest that you maybe convert this to a block comment since its providing documentation about the prop, but more importantly, can you document where this component will be displayed in the card when defined. The name of the prop and this comment here really provides no indication to a consumer of it (Developer) on where this prop's value (A component) will be rendered.

Also - Maybe rename it to be more explicit ?  `Decorator` is very generic :) 

_(same comment for the other variations of the card)_

Reply:
thanks for the suggestion, updated the jsdoc 81ee0c0a7a4fb1642915580128ad7026a6771d67

although, for today my brain is finished so no better idea for the naming. do you maybe have any suggestion? (for that, i'll open a follow-up pr)